index.bs

```diff
@@ -1473,18 +1468,19 @@ partial interface MLGraphBuilder {
 <div dfn-for="MLGraphBuilder/argMin(input, options), MLGraphBuilder/argMax(input, options)" dfn-type=argument>
```

Comment:
Add the axis argument here, or you'll get lint errors. (Bikeshed won't flag this, unfortunately.)